packages/react-ui-core/src/Form/Label.js

```diff
@@ -35,7 +35,7 @@ export default class Label extends Component {
 
     return (
       /* eslint-disable jsx-a11y/label-has-for */
-      <label {...props} className={classnames}>
+      <label data-tid="label" {...props} className={classnames}>
```

Comment:
Might be easier to read if the props were on separate lines (max for linter is 4 however), so this is up to you. 👍

Reply:
Ohk.